### ISSUE TITLE
Message improvements don't working on submod

### ISSUE THREAD

**Issue by Emefar** [DinoDevs/GladiatusCrazyAddon#116]
Show links, guilds info do not working on submod 'messageNew'

When i click show previous message button: 
![image](https://user-images.githubusercontent.com/36764954/48724287-e2f81400-ec39-11e8-8709-87f3add2dfda.png)

**Comment by GramThanos** [comment]
Yeah, we forgot to implement it there too when the previous messages button was added. Guild info should be added.

Links on private messages are not parsed for security reasons.
But, we can flag the message as "trusted" if it was send by a guild mate.

**Comment by FrutyX** [comment]
@GramThanos Well, well, well, what do I see.

A "bug" label and an unfinished feature, since 2018, absolutely shameless! The reckoning day had to come, Gram.

I currently can't test it, should I? Was it patched?

**Comment by FrutyX** [comment]
Hello!

This Issue has been selected to be merged into our Feature Requests & Enhancements overview Issue -  #530.

Please note that this does not change anything, the request is still taken in account, but it's moved for better and cleaner GitHub management.

Thank you for your feedback and ideas! We appreciate them :).
